execution/contexts/call_context.go

```diff
@@ -183,11 +183,18 @@ func (ctx *CallContext) Deliver(inAcc, outAcc *acm.Account, value uint64) error
 					"caller_address", inAcc.GetAddress(),
 					"callee_address", ctx.tx.Address)
 			} else {
-				exception = errors.ErrorCodef(errors.ErrorCodeInvalidAddress,
-					"CallTx to an address (%v) that holds no code", ctx.tx.Address)
-				ctx.Logger.Info.Log(exception,
-					"caller_address", inAcc.GetAddress(),
-					"callee_address", ctx.tx.Address)
+				err := inAcc.SubtractFromBalance(value)
```

Comment:
vm.Call already transfers. Instead we need to not abort above when calling an empty contract.